events/guildMemberRemove.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
         const embed = new EmbedBuilder()
             .setColor(DARK_RED)
             .setTitle(`Membre parti`)
-            .setDescription(`<@${member.id}>`)
+            .setDescription(member)
```

Comment:
Il faut que ce soit une string

```suggestion
            .setDescription(`${member}`)
```

Reply:
Plutôt utiliser toString() directement dans ce cas la, ça fera une opération en moins

```suggestion
            .setDescription(member.toString())
```